Raven.Tests.Issues/RavenDB_4836.cs

```diff
@@ -0,0 +1,112 @@
+﻿using System.Collections.Generic;
+using System.ComponentModel.Composition.Hosting;
+using System.Linq;
+using Lucene.Net.Analysis;
+using Lucene.Net.Analysis.BR;
+using Lucene.Net.Documents;
+using Raven.Abstractions.Data;
+using Raven.Abstractions.Indexing;
+using Raven.Client.Bundles.MoreLikeThis;
+using Raven.Client.Embedded;
+using Raven.Client.Indexes;
+using Raven.Database.Indexing;
+using Raven.Database.Plugins;
+using Raven.Tests.Helpers;
+using Xunit;
+
+namespace Raven.Tests.Issues
+{
+    public class RavenDB_4836 : RavenTestBase
+    {
+        private readonly EmbeddableDocumentStore _documentStore;
+
+        public RavenDB_4836()
+        {
+            _documentStore = NewDocumentStore();
```

Comment:
Just for the record: RavenTestBase [disposes](https://github.com/ayende/ravendb/blob/v3.5/Raven.Tests.Helpers/RavenTestBase.cs#L911) all document stores created using [NewDocumentStore()](https://github.com/ayende/ravendb/blob/v3.5/Raven.Tests.Helpers/RavenTestBase.cs#L270).